ext/standard/tests/file/bug28790.cache.phpt

```diff
@@ -0,0 +1,53 @@
+--TEST--
+Bug #28790: Add php.ini option to disable stat cache (with cache)
+--CREDITS--
+jnoll <jnoll [at] prim [dot] hu>
+Kevin Lyda <kevin [at] lyda [dot] ie>
+--FILE--
+<?php
+
+    $php = '"'.getenv('TEST_PHP_EXECUTABLE').'"';
```

Comment:
Please de-indent PHP files.

Reply:
I think this is now addressed.

Also submitted #7450 to fix the remaining `*.phpt` indents.